docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/psa-config-template.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Pod Security Admissions (PSA) Configuration Templates 
+---
+
+[Pod Security admission (PSA)](./pod-security-standards.md) configuration templates are a Rancher custom-defined resource (CRD), available in Rancher v2.7.2 and above. The templates provide pre-defined security configurations that you can apply to a cluster:
+
+- `rancher-privileged`: The most permissive configuration. It doesn't restrict the behavior of any pods. This allows for known privilege escalations. This policy has no exemptions.
+- `rancher-restricted`: A heavily restricted configuration that follows current best practices for hardening pods. You must make [namespace-level exemptions](./pod-security-standards.md#rancher-on-psa-restricted-clusters) for Rancher components.
+
+## Assign a Pod Security Admissions (PSA) Configuration Template
+
+You can assign a PSA template at the same time that you create a downstream cluster. You can also add a template by configuring an existing cluster.
+
+### Assign a Template During Cluster Creation 
+
+For RKE2 and K3s:
+
+1. In the upper left corner, click **☰ > Cluster Management**.
+1. On the **Clusters** page, click the **Create** button.
+1. Select a provider.
+1. On the **Cluster: Create** page, go to **Basics > Security**. 
+1. In the **Default Pod Security Admission** dropdown menu, select the template you want to assign.
+
+For RKE1:
+
+1. In the upper left corner, click **☰ > Cluster Management**.
+1. On the **Clusters** page, click the **Create** button.
+1. Select a provider.
+1. On the **Add Cluster** page, under **Cluster Options**, click **Advanced Options**.
+1. In the **Pod Security Admission Configuration Template** dropdown menu, select the template you want to assign. 
+
+### Assign a Template to an Existing Cluster
+
+For RKE2 and K3s:
+
+1. In the upper left corner, click **☰ > Cluster Management**.
+1. Find the cluster you want to update in the **Clusters** table, and click the **⋮**.
+1. Select **Edit Config** .
+1. In the **Default Pod Security Admission** dropdown menu, select the template you want to assign.
+
+For RKE1:
+
+1. In the upper left corner, click **☰ > Cluster Management**.
+1. Find the cluster you want to update in the **Clusters** table, and click the **⋮**.
+1. On the **Add Cluster** page, go to **Cluster Options > Advanced Options**.
+1. In the **Pod Security Admission Configuration Template**, select the template you want to assign. 
+
+### Hardening the Cluster
```

Comment:
```suggestion
### Hardening an RKE2 Cluster
```

Reply:
The above is a suggestion, because the CIS profile option is only applicable to RKE2 clusters. Otherwise we must let this clear in some other way.